subsys/testsuite/ztest/src/ztest_new.c

```diff
@@ -464,12 +476,51 @@ struct ztest_unit_test *ztest_get_next_test(const char *suite, struct ztest_unit
 	return NULL;
 }
 
+#if (IS_ENABLED(CONFIG_ZTEST_DO_THE_SHUFFLE))
```

Comment:
I think you can remove this `if` and just always have these functions available

Reply:
The use of rand and srand in these functions need these to be config dependent.